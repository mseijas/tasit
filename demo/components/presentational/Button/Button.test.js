import React from "react";
import { shallow } from "enzyme";
import Button from "./Button";

describe("Button", () => {
  it("renders the component", async () => {
    const onPress = () => {};
    expect(
      shallow(<Button title="Title" onPress={onPress} />)
    ).toMatchSnapshot();
  });
});
