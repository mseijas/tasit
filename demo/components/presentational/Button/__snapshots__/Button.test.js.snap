// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders the component 1`] = `
<Button
  activeOpacity={1}
  containerStyle={
    Object {
      "backgroundColor": "#2196f3",
      "borderRadius": 3,
      "overflow": "hidden",
      "padding": 20.009999999999998,
    }
  }
  onPress={[Function]}
  style={
    Object {
      "color": "white",
      "fontSize": 27.53633962602873,
    }
  }
>
  TITLE
</Button>
`;
